src/main/java/io/github/opencubicchunks/cubicchunks/core/network/WorldEncoder.java

```diff
@@ -51,6 +51,7 @@ static void encodeCubes(PacketBuffer out, Collection<Cube> cubes) {
         cubes.forEach(cube -> {
             out.writeBoolean(cube.isEmpty());
             out.writeBoolean(cube.getStorage() != null);
+            out.writeBoolean(cube.getBiomeArray() != null);
```

Comment:
This is network-incompatible change. Either make it network-compatible, or set the right required remote versions. And make sure to test it with non-dev builds, both when trying to join on older client, and when trying to join older server.